vibi-dpu/src/bitbucket/reviewer.rs

```diff
@@ -0,0 +1,156 @@
+use reqwest::{Response, header::{HeaderMap, HeaderValue}};
+use serde::Serialize;
+use serde_json::Value;
+use reqwest::Error;
+use std::io;
+
+use crate::db::user::get_workspace_user_from_db;
+use crate::utils::review::Review;
+use crate::utils::user::BitbucketUser;
+
+use super::{config::{bitbucket_base_url, get_client, prepare_headers}};
+
+pub async fn add_reviewers(user_key: &str, review: &Review, access_token: &str) {
+    let url = prepare_add_reviewers_url(review.repo_owner(), review.repo_name(), review.id());
```

Comment:
This function is not correctly named. It only gets the pr info, doesn't add reviewers